lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
@KLVTZ Absolutely looking good for getting this merged (sorry, just back from a 3 week vacation, and have been quite unplugged). Would like to see @M4v3R's response to your comment though.